### PR TITLE
circleci: Add configs for 9P

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: configs/generic_config.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # vfat filesystem:
+# TODO: vfat is going to be deprecated for testing in favour of 9p
```
Comment (hugelgupf) [inline configs/generic_config.txt]: remove this

we still want vfat for testing diskbooting, e.g.

**Comment by rminnich** [comment]
well, somebody is gonna have to explain to me why this is a good idea, b/c I don't see it.

File: configs/generic_config.txt (hunk 1 above)
Comment (rjoleary) [reply configs/generic_config.txt]: done

**Comment by hugelgupf** [comment]
Ron, it's not that this is faster or slower. Nobody cares about the speed in *tests*. They care that files are writable, and qemu's vfat emulation sucks.

**Pull request merged**